sdkconfig.defaults

```diff
@@ -137,14 +140,20 @@ CONFIG_SHA_BADGE_EINK_LUT_DEBUG=
 CONFIG_SHA_BADGE_POWER_DEBUG=
 
 #
-# SHA2017 Badge First Run
+# Badge First Run
 #
 CONFIG_DEBUG_ADD_DELAYS=
 CONFIG_BASVS_BROKEN_BADGE=
 CONFIG_ALLOW_BATTERY_FIRST_BOOT=
 
 #
-# SHA2017 BPP
+# OTA firmware update
+#
+CONFIG_OTA_SERVER_NAME="badge.sha2017.org"
```

Comment:
kan ook later `badge.team` worden ;)

Reply:
dit is slechts verplaatsen van het blokje. ik heb alleen de default config opnieuw gegenereerd. geen andere wijzigingen.

Reply:
check . .